mods/d2k/rules/infantry.yaml

```diff
@@ -119,8 +125,7 @@ fremen:
 	Health:
 		HP: 700
 	RevealsShroud:
-		Range: 7c0
-	Passenger:
```

Comment:
I took a look, and it's actually not. Passenger is inherited from `^Infantry`, I just forgot to make this a separate commit or mention it.